Ownership/AC - Central/12.txt

```diff
@@ -48,4 +48,12 @@ ALTOWNER:Observing LON_E_CTR:LGS:LEN:O:LMEN:LE:LC:LSC:L
 ALTOWNER:Observing LON_C_CTR:LGS:LEN:LE:O:LMEN:LC:LSC:L
 ALTOWNER:Observing LON_SC_CTR:LGS:LEN:LE:LC:O:LMEN:LSC:L
 ALTOWNER:Observing London FIR:LGS:O:LMEN:LEN:LE:LC:LSC:L
-BORDER:LE01:LTC36:LTC60:LTC61:LTC62:LC11b:LED05a
+BORDER:LE01:LTC36:LTC60:LTC61:LTC62:LC11b:LED07
+
+SECTOR:London S12 03 (195-660):19500:66000
```

Comment:
I can't see a base FL195 block in the AC Clacton diagram? It all seems to be 215 or higher.

Reply:
Good spot. I believe this is an issue with the diagrams, rather than the defintion here.

![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/13762210/ef536de7-6654-4822-93d3-bac7628c9aea)

@hazzas-99 or @KHardern to confirm